python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1484,6 +1486,26 @@ keys:
                       - str
                     min: 2
                     max: 64
+          evpn_ethernet_segment:
+            type: list
+            primary_key: domain
+            items:
+              type: dict
+              keys:
+                domain:
+                  type: str
+                  valid_values:
+                    - "all"
+                    - "local"
+                    - "remote"
+                identifier:
+                  type: str
+                  description: "10 octet ethernet segment identifier."
+                  required: true
```

Comment:
Are these really required? You have `if` around it in the templates.

Reply:
I mean, it won't work without them, but the cli will happily just accept something like
```bash
address-family evpn
   evpn ethernet-segment domain X
   !
```

Reply:
I would prefer to remove the required from the schema and put back the checks in the template.